app/views/course/material/folders/_folder.slim

```diff
@@ -8,6 +11,11 @@
       = format_html(folder.description)
 
   td = format_datetime(folder.updated_at)
+  td
+    - if can?(:edit, folder) && show_sdl_warning?(folder)
+      span title=t('.visible_because_sdl')
+       => fa_icon 'eye'.freeze
+    = format_datetime(folder.start_at)
```

Comment:
Do you think it is better to show or hide this to/from students?

Reply:
Yeah maybe can just hide the whole column from students, they don't need to know anyway.

Either they see the whole folder or they don't.